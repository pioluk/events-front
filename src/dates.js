// @flow

import fecha from 'fecha'

export const formatDateTimeShort = (date: string) =>
  fecha.format(new Date(date), 'MMM D, HH:mm')
