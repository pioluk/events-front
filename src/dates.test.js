// global describe,it,expect

import { formatDateTimeShort } from './dates'

describe('dates', () => {
  describe('formatDateTimeShort', () => {
    it('should exist', () => {
      expect(formatDateTimeShort).toBeInstanceOf(Function)
    })

    it('should format date correctly', () => {
      const formattedString = formatDateTimeShort('2016-01-01T12:00:00.000Z')
      expect(formattedString).toMatchSnapshot()
    })
  })
})
