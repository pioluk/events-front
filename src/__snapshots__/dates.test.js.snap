exports[`dates formatDateTimeShort should format date correctly 1`] = `"Jan 1, 13:00"`;
